packages/taquito-tzip16/src/viewKind/michelson-storage-view.ts

```diff
@@ -144,52 +145,22 @@ export class MichelsonStorageView implements View {
     this.findForbiddenInstructionInViewCode(this.code);
     this.illegalUseOfSelfInstruction(this.code);
 
-    const { arg, viewParameterType } = this.formatArgsAndParameter(args);
-
-    const storageType: any = this.contract.script.code.find((x: any) => x.prim === 'storage');
-    const storageArgs = storageType.args[0];
+    const { arg } = this.formatArgsAndParameter(args);
 
     // currentContext
     const storageValue: any = await this.readProvider.getStorage(this.contract.address, 'head');
     const chainId = await this.readProvider.getChainId();
-    const contractBalance = (
-      await this.readProvider.getBalance(this.contract.address, 'head')
-    ).toString();
-    const blockTimestamp = await this.readProvider.getBlockTimestamp('head');
-
-    const code = this.adaptViewCodeToContext(this.code, contractBalance, blockTimestamp, chainId);
```

Comment:
I am not convinced that we can use `runScriptView` for the tzip16 view. The code property is removed here but is essential to execute the view as it tells what the instructions are. This kind of view is different from the on-chain view, the instructions are stored off-chain.

The integration test throws: 
 [{"kind":"permanent","id":"proto.014-PtKathma.viewNotFound","contract":"KT1RVLD2t2PG8CwC9NynAbgVoYxddNrGY7NV","view":"GetCounter"}]
 
 Seems like we need to go back to the old implementation for this file.